src/Message/RestCreateWebhookRequest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Omnipay\PayPal\Message;
+
+/**
+ * @final
+ */
+class RestCreateWebhookRequest extends AbstractRestRequest
```

Comment:
why is this class not final?

Reply:
Was done this way in order to be able to mock it in the unit test of the implementing repository where this was built in. But this is actually not an issue anymore, so I'll just make it final using the keyword.